Index: .idea/sonarlint/securityhotspotstore/index.pb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n7\n\u0007pom.xml\u0012,4\\4\\442292b8a7efeabbe4cc176709b833b1792140ec\n`\n0src/main/java/demo01/Abstractclasses/Animal.java\u0012,b\\a\\ba890c5515e9f71bb095f6f28276c64fa625b677\n]\n-src/main/java/demo01/Abstractclasses/Dog.java\u0012,6\\1\\61d8082579c28ef6fb7eb08236f5647d739960a6\n[\n+src/main/java/demo01/interfaces/Animal.java\u0012,6\\c\\6c1c99afb4b786b4ccb88dd31a87983f56fbf6b3\n^\n.src/main/java/demo01/interfaces/MammalInt.java\u0012,f\\b\\fbc321b2c8f6a1a3e9f899546178cf9e521b5154\nI\n\u0019src/main/java/demo01/biji\u0012,e\\3\\e3e54b549738d90e6f158cbeb2355fdecb09c05f\nb\n2src/main/java/demo01/interfaces/interfaceTest.java\u0012,7\\1\\71199fc2bcb1f917f25c2e4e32df8e90fbec1455
===================================================================
diff --git a/.idea/sonarlint/securityhotspotstore/index.pb b/.idea/sonarlint/securityhotspotstore/index.pb
--- a/.idea/sonarlint/securityhotspotstore/index.pb	
+++ b/.idea/sonarlint/securityhotspotstore/index.pb	
@@ -12,4 +12,6 @@
 I
 src/main/java/demo01/biji,e\3\e3e54b549738d90e6f158cbeb2355fdecb09c05f
 b
-2src/main/java/demo01/interfaces/interfaceTest.java,7\1\71199fc2bcb1f917f25c2e4e32df8e90fbec1455
\ No newline at end of file
+2src/main/java/demo01/interfaces/interfaceTest.java,7\1\71199fc2bcb1f917f25c2e4e32df8e90fbec1455
+`
+0src/main/java/demo01/Abstractclasses/people.java,b\e\be3e4e9c147972aa45dae24ee70f58f652c8fb34
\ No newline at end of file
Index: .idea/sonarlint/issuestore/index.pb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n7\n\u0007pom.xml\u0012,4\\4\\442292b8a7efeabbe4cc176709b833b1792140ec\n`\n0src/main/java/demo01/Abstractclasses/Animal.java\u0012,b\\a\\ba890c5515e9f71bb095f6f28276c64fa625b677\n]\n-src/main/java/demo01/Abstractclasses/Dog.java\u0012,6\\1\\61d8082579c28ef6fb7eb08236f5647d739960a6\n[\n+src/main/java/demo01/interfaces/Animal.java\u0012,6\\c\\6c1c99afb4b786b4ccb88dd31a87983f56fbf6b3\n^\n.src/main/java/demo01/interfaces/MammalInt.java\u0012,f\\b\\fbc321b2c8f6a1a3e9f899546178cf9e521b5154\nI\n\u0019src/main/java/demo01/biji\u0012,e\\3\\e3e54b549738d90e6f158cbeb2355fdecb09c05f\nb\n2src/main/java/demo01/interfaces/interfaceTest.java\u0012,7\\1\\71199fc2bcb1f917f25c2e4e32df8e90fbec1455
===================================================================
diff --git a/.idea/sonarlint/issuestore/index.pb b/.idea/sonarlint/issuestore/index.pb
--- a/.idea/sonarlint/issuestore/index.pb	
+++ b/.idea/sonarlint/issuestore/index.pb	
@@ -12,4 +12,6 @@
 I
 src/main/java/demo01/biji,e\3\e3e54b549738d90e6f158cbeb2355fdecb09c05f
 b
-2src/main/java/demo01/interfaces/interfaceTest.java,7\1\71199fc2bcb1f917f25c2e4e32df8e90fbec1455
\ No newline at end of file
+2src/main/java/demo01/interfaces/interfaceTest.java,7\1\71199fc2bcb1f917f25c2e4e32df8e90fbec1455
+`
+0src/main/java/demo01/Abstractclasses/people.java,b\e\be3e4e9c147972aa45dae24ee70f58f652c8fb34
\ No newline at end of file
